edx_proctoring/static/proctoring/js/views/proctored_exam_allowance_view.js

```diff
@@ -10,7 +10,8 @@ edx = edx || {};
         initialize: function() {
             this.allowance_types = [
                 ['additional_time_granted', gettext('Additional Time (minutes)')],
-                ['review_policy_exception', gettext('Review Policy Exception')]
+                ['review_policy_exception', gettext('Review Policy Exception')],
```

Comment:
You want this view to have the waffle flag injected into it.
Then you can use the waffle in this view to determine the `add_allowance_view` to use for the pop up.